examples/keyboard_emu/keyboard_emu.cpp

```diff
@@ -0,0 +1,189 @@
+//This example uses pin 40, 38, and 37.
```

Comment:
Maybe add a short comment that elaborates about the example program. I also use following block comments style for header comments

/*
*
*/

I would like to keep the style the same, can you change it to that style?.